packages/dev-utils/src/devApp/render.tsx

```diff
@@ -128,9 +132,34 @@ class DevAppBuilder {
    * Build a DevApp component using the resources registered so far
    */
   build(): ComponentType<{}> {
+    const dummyPlugin = createPlugin({
```

Comment:
Let's use this method instead of creating a dummy plugin + extension imo: https://github.com/backstage/backstage/blob/f7b951d9f5cfaf90eb5d228888a41ab100e52fdb/packages/test-utils/src/testUtils/appWrappers.tsx#L116

Reply:
Awesome, there it was! The shortcut! :) Fixed.